custom_components/test/fuzzing/fuzzer_overview.puml

```diff
@@ -48,20 +48,26 @@ class "GeneratorFuzzer" as GFuzzer << class >> {
     + fuzz(self, ???) : list
 }
 '''''''''''''''''''''''''''''''''''''''
-class "GrammarFuzzer" as GrFuzzer << class >> {
+class "CostGrammarType" as CGType << class >> {
```

Comment:
I am not sure if the `CostGrammarType` class is correctly inserted into the UML. I have not found an instance of the class in the code.